Resolved Bug 522 : Storybook > Charts > Scatter Chart > Docs > Scatter Chart Icon Does Not Display Proper

suchithravipin1991 opened the pull request:
## Description
> bug id :  #651- Design System > Elements > alert > With icon > Icon stroke field > Icon stroke field is not working #651-

   bug id : #522-Storybook >Charts>Scatter Chart>Docs> Scatter Chart icon does not display proper.

Bugs Resolved :
bug id :  #651 : Icon stroke is no longer needed So commented out the property stroke
bug id : #522 : Chart icon displaying properly  

## Screenshots :

 
 
![icon stroke removed](https://github.com/user-attachments/assets/3245e0ee-adeb-4666-a4dc-561d4842a76e)
![scatter](https://github.com/user-attachments/assets/057db572-bd9d-4c8d-b9c4-7783d2136c76)

amarphule reviewed (approved):
Reviewed changes and approved

suchithravipin1991 committed:
bug id: 651 & 522 resolved

bug id :  #651- Design System > Elements > alert > With icon > Icon stroke field > Icon stroke field is not working #651-

bug id : #522-Storybook >Charts>Scatter Chart>Docs> Scatter Chart icon does not display proper.

suchithravipin1991 committed:
Bugs resolved

## Description
> bug id :  #651- Design System > Elements > alert > With icon > Icon stroke field > Icon stroke field is not working #651-

   bug id : #522-Storybook >Charts>Scatter Chart>Docs> Scatter Chart icon does not display proper.

Bugs Resolved :
bug id :  #651 : Icon stroke is no longer needed So commented out the property stroke
bug id : #522 : Chart icon displaying properly

## Screenshots :

![icon stroke removed](https://github.com/user-attachments/assets/3245e0ee-adeb-4666-a4dc-561d4842a76e)
![scatter](https://github.com/user-attachments/assets/057db572-bd9d-4c8d-b9c4-7783d2136c76)

Ketan-Khalate commented on raaghu-elements/src/rds-alert/rds-alert.tsx:
```diff
@@ -64,7 +66,7 @@ const RdsAlert = (props: RdsAlertProps) => {
                             name={props.icon || " "}
                             fill={props.iconFill}
-                            stroke={props.iconStroke}
+                           // stroke={props.iconStroke}
```
@suchithravipin1991 here commented line is a input for that element we cannot comment it out just make iconStroke "false" from alert.stories file and check your changes

suchithravipin1991 committed:
scatter chart icon issue resolved & updated

Bug ID: 522scatter chart icon issue resolved & updated

Ketan-Khalate committed:
Merge branch 'development' into users/suchithra.aa/bug-storybook-scatter-chart-icon-issue

Ketan-Khalate commented:
Changes Reviewed and PR Approved
